packs/data/feats.db/chronoskimmer-dedication.json

```
@@ -24,7 +24,65 @@
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "initiative",
+                "key": "RollOption",
+                "label": "Chronoskimmer Stabilize",
+                "option": "chronoskimmer-stabilize",
+                "toggleable": true,
+                "value": false
+            },
+            {
+                "domain": "initiative",
+                "key": "RollOption",
+                "label": "Chronoskimmer Destabilize (Success)",
+                "option": "chronoskimmer-destabilize-success",
+                "toggleable": true,
+                "value": false
+            },
+            {
+                "domain": "initiative",
+                "key": "RollOption",
+                "label": "Chronoskimmer Destabilize (Fail)",
+                "option": "chronoskimmer-destabilize-fail",
+                "toggleable": true,
+                "value": false
+            },
+            {
```

```suggestion
        "rules": [
            {
```
Just get rid of them, roll substitutions maybe make sense to live in the roll dialog.